boards/arduino/arduino_opta/Kconfig.arduino_opta

```
@@ -0,0 +1,6 @@
+# Copyright (c) 2023 Felipe Neves
+# SPDX-License-Identifier: Apache-2.0
+
+config BOARD_ARDUINO_OPTA_STM32H747XX_M4
```

```suggestion
config BOARD_ARDUINO_OPTA
```

Please use the pure board name identifier, as that will ensure the correct soc is still selected if variants are later added to the board.